meta-openbmc-machines/meta-openpower/meta-ibm/meta-witherspoon/recipes-kernel/linux/linux-obmc_4.7.bb

```diff
@@ -0,0 +1,6 @@
+KBRANCH ?= "dev-4.7"
+LINUX_VERSION ?= "4.7"
+SRCREV="05543e532c78185ba82057415f96d13e5f500072"
```

Comment:
We should make sure @shenki is aware of this.  Sometimes he does a rebase / push--force, so we probably want him to get this into the openbmc tree and add a tag to point at.